cmd/vaRequirement.go

```diff
@@ -25,7 +28,14 @@ var vaRequirementListCmd = &cobra.Command{
 			$ vf-admin va requirement list <va-id>
 	`),
 	Args: cobra.ExactArgs(1),
-	RunE: list.CmdRunE,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var op list.HTTPOperation
+
+		op.SetRequestURLArguments(args)
```

Comment:
You should handle the error here and in other places.

Reply:
Wasn't sure if that's needed because for the ops I added, `SetRequestURLArguments` never returns an error cause the IDs are exptected to be strings. I also can't change the function signature itself to not return an error since that's defined in the HTTPOperation interface

Reply:
Yeah, I can see that it returns no error but it should always be a good idea to still catch an error in case we change the function body. Then when we do change the function body, we will already have the code working.

Reply:
@gesarki I've already merged into `main` but you should create a branch to fix this. It's still popping up in other files.

e.g. vaTimeslot.go